assets/app/styles/_list.less

```diff
@@ -0,0 +1,22 @@
+.confirm-delete {
```

Comment:
same with this one, we'll probably reuse this, call it .inline-confirmation and move to _core.less